ITEST/itest.tracing/pom.xml

```diff
@@ -32,6 +32,13 @@
                             <webservice-webapp>
                                 ../../WSFrontend/org.openl.rules.ruleservice.ws.all/target/webapp
                             </webservice-webapp>
+                            <otel.javaagent.extensions>../../WSFrontend/org.openl.rules.ruleservice.ws.tracing.opentelemetry/target/openl-opentelemetry-extension-5.26.6-SNAPSHOT.jar</otel.javaagent.extensions>
+                            <otel.instrumentation.methods.include>org.openl.rules.calc.Spreadsheet[innerInvoke];org.openl.rules.dt.DecisionTable[innerInvoke];org.openl.rules.tbasic.Algorithm[innerInvoke];org.openl.rules.method.table.TableMethod[innerInvoke];org.openl.rules.cmatch.ColumnMatch[innerInvoke]</otel.instrumentation.methods.include>
+                            <otel.metrics.exporter>none</otel.metrics.exporter>
+                            <otel.service.name>RulerService-Test</otel.service.name>
+                            <otel.traces.exporter>logging-otlp</otel.traces.exporter>
+                            <otel.logs.exporter>logging-otlp</otel.logs.exporter>
+                            <io.opentelemetry.javaagent.slf4j.simpleLogger.logFile>./target/opentelemetry.log</io.opentelemetry.javaagent.slf4j.simpleLogger.logFile>
```

Comment:
it's better to use `${project.build.directory}` property instead of hardcoding `./target`